bitnami/postgresql/values.yaml

```diff
@@ -1099,6 +1099,11 @@ backup:
     ttlSecondsAfterFinished: ""
     ## @param backup.cronjob.restartPolicy Set the cronjob parameter restartPolicy
     restartPolicy: OnFailure
+    ## @param backup.cronjob.podSecurityContext.enabled Enable PodSecurityContext for CronJob/Backup
+    ## @param backup.cronjob.podSecurityContext.fsGroup Group ID for the CronJob
+    podSecurityContext:
+      enabled: false
```

Comment:
Why are you setting this to `false` by default? Is it for backwards compatibility? Do you have any use case where setting it to `true` could be problematic?

Reply:
I don't think it's a problem to have it enabled by default. In fact the CronJob didn't even work for me, when this setting weren't set. 

I will enable it by default.